src/api/app/mailers/daily_email_mailer.rb

```diff
@@ -0,0 +1,36 @@
+class DailyEmailMailer < ActionMailer::Base
+  helper :comment
+
+  def set_headers
+    @host = ::Configuration.obs_url
+    @configuration = ::Configuration.first
+
+    headers['Precedence'] = 'bulk'
+    headers['X-Mailer'] = 'OBS Notification System'
+    headers['X-OBS-URL'] = ActionDispatch::Http::URL.url_for(controller: :main, action: :index, only_path: false, host: @host)
+    headers['Auto-Submitted'] = 'auto-generated'
+    headers['Return-Path'] = mail_sender
+    headers['Sender'] = mail_sender
+  end
+
+  def mail_sender
+    'OBS Notification <' + ::Configuration.admin_email + '>'
+  end
```

Comment:
Lines 2 - 18 are basically just c&p from app/mailers/event_mailer.rb. Can we maybe move it to a common parent class?